modules/virt-template-vmi-probe-config.adoc

```diff
@@ -3,12 +3,12 @@
 // * virt/logging_events_monitoring/virt-monitoring-vm-health.adoc
 
 [id="virt-template-vmi-probe-config_{context}"]
-= Template: Virtual machine instance configuration file for defining health checks
+= Template: Virtual machine configuration file for defining health checks
```

Comment:
I think it would make sense to change the module ID and file name to reflect the title change. After changing the file name, make sure to correct the module reference in the assembly.

Same comment applies to the `virt-template-vmi-pxe-config.adoc` and `virt-template-windows-vmi.adoc` modules.